Helper/Cart.php

```diff
@@ -474,21 +474,21 @@ public function getCartData($payment_only, $place_order_payload, $quote = null)
             $place_order_payload = json_decode($place_order_payload);
 
             $email                = @$place_order_payload->email ?: $email;
-            $billingAddress       = @$place_order_payload->billingAddress;
-            $billingStreetAddress = (array)@$billingAddress->street;
+            $billing_address      = @$place_order_payload->billingAddress;
```

Comment:
nit: let's stick to camelCase (billingAddress) per magento's convention.

Reply:
done for the whole file